scripts/get-all-test-paths.sh

```diff
@@ -6,10 +6,10 @@ declare -a array1=( "tests/unit/*.py" "tests/integration/*.py" "tests/distribute
 declare -a array2=( $(ls -d tests/{unit,integration,distributed}/*/ | grep -v '__pycache__' ))
 dest1=( "${array1[@]}" "${array2[@]}" )
 
-declare -a array1=( "tests/daemon/unit/*.py" "tests/daemon/integration/*.py" )
+declare -a array1=( "tests/daemon/unit/*.py" )
```

Comment:
I would leave it to make sure if we add them that they run

Reply:
Hmm I am not convinced. This creates a github action for each time we run the tests which uses quite some computation power. Even thought we can use as much github-actions as we like, since we are open source, I'd at least not run meaningless commands the whole time. But that is just my opinion and I can add it back.

Reply:
are u sure it creates a gthub action it not found?

Reply:
if it does create the action yes I would remove

Reply:
Yes it creates the action.

Reply:
then kill it heavily